packages/cast/blockchain-driver-eth/src/transaction-info-provider/node/EthereumNodeTransactionInfoProvider.ts

```diff
@@ -34,6 +35,10 @@ export class EthereumNodeTransactionInfoProvider
       );
       return undefined;
     }
+
+    const maxFeePerGas = formatParameter(transaction.maxFeePerGas);
+    const maxPriorityFeePerGas = formatParameter(transaction.maxPriorityFeePerGas);;
```

Comment:
Je crois tu peux ajouter le type des deux consts:
    
```
    const maxFeePerGas:number = formatParameter(transaction.maxFeePerGas);
    const maxPriorityFeePerGas:number = formatParameter(transaction.maxPriorityFeePerGas);
```

Reply:
I moved the function calls directly into the values of the constructed object.